src/store/statistics.js

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ * Copyright (c) 2019-present for NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License ");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import Lock from './lock'
+import { NetworkService, StatService } from '../infrastructure'
+
+const LOCK = Lock.create()
+
+export default {
+  namespaced: true,
+  state: {
+    // If the state has been initialized.
+    initialized: false,
+    loading: false,
+    loadingInfo: false,
+    loadingBlockTimeDifference: false,
+    loadingTransactionPerBlock: false,
+    error: false,
+    networkTransactionFees: [],
+    networkRentalFees: [],
+    blockTimeDifferenceData: [],
+    transactionPerBlockData: []
```

Comment:
Can you change this please to the `Data Manager` approach?

Reply:
applied Data manager approach in refactor infra branch as well :)